verilog/analysis/checkers/always_comb_blocking_rule.cc

```diff
@@ -68,7 +68,7 @@ void AlwaysCombBlockingRule::HandleSymbol(const verible::Symbol& symbol,
           *node, NodeEnum::kNonblockingAssignmentStatement, 1);
 
       if (leaf.get().token_enum == TK_LE)
-        violations_.push_back(LintViolation(leaf, kMessage, match.context));
+        violations_.insert(LintViolation(leaf, kMessage, match.context));
```

Comment:
Replace `#include <vector>` with `#include <set>` in these .cc files.

Reply:
Sure, thanks, replaced in most of the files and added both where vectors were still used for internal logic